### DISCUSSION TITLE
Request: vscode settings option to configure linting severity or disable entirely

### DISCUSSION THREAD

**Discussion by vv3rd** [biomejs/biome-vscode#84] [Ideas]
This will allow flexibility in developer workflows, making it easier to distinguish typescript errors from linting errors or using biome linting from command-line without editor hint that can be distracting.

Eslint configuration as an example:

![Screenshot 2024-01-16 at 14 37 35](https://github.com/biomejs/biome/assets/42813112/a789e2ec-0695-4345-ab6a-624453e5fadd)

(Personal note: pls I beg you they are so distracting I can't pay attention to every lint error when prototyping, but I need formatter for quicker editing)

**Comment by mbalc** [comment]
bump, I beg you as well, it's unusable in the current state

![image](https://github.com/biomejs/biome-vscode/assets/11033790/f6d47c13-094e-4afd-aa74-0381145e34b9)

all of those should be info, or warning at most

**Comment by ematipico** [comment]
I suppose you can't use a configuration file?
